modules/framework/galasa-parent/dev.galasa.framework.api.users/src/test/java/dev/galasa/framework/api/users/internal/routes/UsersDeleteRouteTest.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.users.internal.routes;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.ServletOutputStream;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.Test;
+
+import dev.galasa.framework.api.common.BaseServletTest;
+import dev.galasa.framework.api.common.EnvironmentVariables;
+import dev.galasa.framework.api.common.HttpMethod;
+import dev.galasa.framework.api.common.InternalUser;
+import dev.galasa.framework.api.common.mocks.MockEnvironment;
+import dev.galasa.framework.api.common.mocks.MockFramework;
+import dev.galasa.framework.api.common.mocks.MockHttpServletRequest;
+import dev.galasa.framework.api.common.mocks.MockHttpServletResponse;
+import dev.galasa.framework.api.common.mocks.MockTimeService;
+import dev.galasa.framework.api.users.mocks.MockUsersServlet;
+import dev.galasa.framework.auth.spi.internal.AuthService;
+import dev.galasa.framework.auth.spi.mocks.MockAuthStoreService;
+import dev.galasa.framework.auth.spi.mocks.MockDexGrpcClient;
+import dev.galasa.framework.auth.spi.mocks.MockFrontEndClient;
+import dev.galasa.framework.auth.spi.mocks.MockUser;
+import dev.galasa.framework.spi.auth.IInternalUser;
+
+public class UsersDeleteRouteTest extends BaseServletTest {
+
+
+    Map<String, String> headerMap = Map.of("Authorization", "Bearer " + BaseServletTest.DUMMY_JWT);
+
+    private static final String path = "\\/([a-zA-Z0-9\\-\\_]+)\\/?";
+    private static final Pattern pattern = Pattern.compile(path);
```

Comment:
We don't want to duplicate the regex pattern in the test - if the pattern in the route changes, then we won't pick up any regex-related errors. We should be grabbing the pattern in the route using `getPathRegex()` in the tests.

See other existing regex tests for inspiration